Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -438,38 +463,40 @@ function Enable-SecureFTPDeploymentForAppServices
                 }
                 else
                 {
-                Write-Host "Do you want to upadate FTP State? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
```

Comment:
If $FTP State is provided, then there is no need for force flag anyway.

Reply:
Resolved as per discussion in the call.